cluster-up/cluster/kind/common.sh

```diff
@@ -102,7 +102,6 @@ function _configure_registry_on_node() {
 
 function _install_cnis {
     _install_cni_plugins
-    _install_calico_cni
```

Comment:
Can the function be deleted now?

Reply:
Yes we can remove all related code, if we would want to use Calico again we have git history

Reply:
Done